Code/Tools/SceneAPI/SceneBuilder/Importers/AssImpImporterUtilities.cpp

```diff
@@ -41,6 +42,32 @@ namespace AZ
                 return boneCount > 0;
             }
 
+            bool AreAllMeshesValid(const aiNode& node, const aiScene& scene)
+            {
+                for (unsigned meshIdx = 0; meshIdx < node.mNumMeshes; ++meshIdx)
+                {
+                    const aiMesh* mesh = scene.mMeshes[node.mMeshes[meshIdx]];
+                    for (unsigned int faceIdx = 0; faceIdx < mesh->mNumFaces; ++faceIdx)
+                    {
+                        aiFace face = mesh->mFaces[faceIdx];
+                        if (face.mNumIndices < 3)
+                        {
+                            // If the mesh has any faces that have less than 3 indices, mark this mesh as invalid.
+                            // This is commonly a control curve object that should be ignored by the mesh importer.
+                            // NOTE: meshes that contain more than 3 vertices are still considered a valid mesh at this step,
+                            // although they should be triangulated already in the assImp import process. Non triangulated meshes
+                            // will be ignored in the later process.
+                            AZ_Printf(Utilities::LogWindow,
```

Comment:
AZ_Warning vs AZ_Printf: This is purely informative. It's quite common to have multiple control curve objects in the maya model. There is nothing "unusual(warning)" or "false(error)" going on in the model at all. There is no action item for the user to do. The justification for the printf is in case there is any valid reason for the user to put in a 2-vertices faced mesh that they actually wanted in the final product - so at least when they search for the missing mesh, they know what's happening behind the scene.

Reply:
For tests we have a backlog item to improve automation around processing scene files. At the moment most of our test coverage comes from https://github.com/o3de/o3de/tree/development/AutomatedTesting/Gem/PythonTests/assetpipeline/fbx_tests
These tests are setup to process an FBX file, and verify the output debug scene graph looks the same as a previously processed one, so we can catch unexpected changes to the output of scene files. You could probably toss one in there that processes a file with a control curve.

Reply:
Thanks for the info @amzn-rhhong and @AMZN-stankowi, much appreciated 👍